pkg/codegen/go/gen_program.go

```diff
@@ -808,13 +809,17 @@ func (g *generator) genLocalVariable(w io.Writer, v *pcl.LocalVariable) {
 			assignment = "="
 		}
 	}
+
```

Comment:
Is it possible to hoist this outside of the switch. It's called that everywhere except for the `!noError` case in `pcl.Invoke` and in the `default:` case. It should be called in the `default:` case.
```suggestion
	if name == "_" {
 		assignment = "="
 	}
```